dpp.rc

```diff
@@ -0,0 +1,32 @@
+1 VERSIONINFO
+ FILEVERSION 10,0,16
+ PRODUCTVERSION 10,0,16
+ FILEFLAGSMASK 0x17L
+#ifdef _DEBUG
+ FILEFLAGS 0x1L
+#else
+ FILEFLAGS 0x0L
+#endif
+ FILEOS 0x4L
+ FILETYPE 0x2L
+ FILESUBTYPE 0x0L
+BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+        BLOCK "040904b0"
+        BEGIN
+            VALUE "CompanyName", "Brainbox.cc"
+            VALUE "FileDescription", "Dpp - The C++ Discord API Library"
+            VALUE "FileVersion", "10.0.16"
+            VALUE "InternalName", "DppD"
```

Comment:
what is DppD?

Reply:
I mark `DppD` as Dpp DLL (yeah internal name only, it's just a name)